apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -489,6 +493,35 @@ export default class ProjectManager {
     }
   }
 
+  private async loadHelper(versionId?: string, isPreview?: boolean) {
```

Comment:
nit: `loadSources` since it returns sources?

Reply:
good call